bricks/dart_frog_dev_server/hooks/pre_gen.dart

```diff
@@ -26,6 +26,7 @@ Future<void> preGen(
   }
 
   reportRouteConflicts(context, configuration);
+  reportRogueRoutes(context, configuration);
```

Comment:
Shouldn't u want to report the rogues first and then the conflicts? Would assume the rogues are warnings, as it doesnt influence the routing itself while the conflicts are truly a problem so you dont want them to "scroll out of screen"

Reply:
Both influence the routing since both conflicts and rogue routes result in unreachable routes for the end user so I would consider the severity to be the same.